### ISSUE TITLE
AttributeError: 'DatabaseWrapper' object has no attribute 'is_mariadb'

### ISSUE THREAD

**Issue by Sanyambansal76** [adamchainz/django-mysql#331]
When i am running the makemigration command for this model

```
from __future__ import unicode_literals

from django.db import models
from django_mysql.models import JSONField

class Owners(models.Model):
    name = models.CharField(max_length=255)
    url = models.CharField(max_length=128, null=True, blank=True)
    email = models.EmailField(null=True, blank=True)
    phone = models.CharField(max_length=32, null=True, blank=True)
    address = models.CharField(max_length=128, null=True, blank=True)
    country = models.CharField(max_length=32, null=True, blank=True)
    description = models.TextField(null=True, blank=True)

class Brands(models.Model):
    name = models.CharField(max_length=255)
    owner = models.ForeignKey(Owners, null=True, blank=True)
    url = models.CharField(max_length=128, null=True, blank=True)
    description = models.TextField(null=True, blank=True)
    tags = JSONField(null=True, blank=True)
```

**I am getting that logs**

```
  File "/home/sanyam/Project/groceryfy/venv/local/lib/python2.7/site-packages/django_mysql/models/fields/json.py", line 65, in _check_mysql_version
    (conn.is_mariadb or conn.mysql_version < (5, 7))
AttributeError: 'DatabaseWrapper' object has no attribute 'is_mariadb'
```

Summary: *summarize the issue here*

*If this is a bug, please fill in the below, else delete*

Django Version: *e.g. Django 1.9*

Database and version used: *e.g. MySQL 5.7*

Version: *e.g. Django-MySQL 1.1.0*

**Comment by adamchainz** [comment]
I suspect you don't have `django_mysql` in your `INSTALLED_APPS`, as previously reported and will be fixed in #323 .

**Comment by adamchainz** [comment]
Refer to https://django-mysql.readthedocs.io/en/latest/installation.html

**Comment by Sanyambansal76** [comment]
thanks it works....

**Comment by shashanks1** [comment]
Probably this error is related to something else, I installed django-mysql from pip and include in my apps. Now, when I am using JSONField in my models and run makemigration then it spit out the error:

```
(conn.is_mariadb or conn.mysql_version < (5, 7))
AttributeError: 'DatabaseWrapper' object has no attribute 'is_mariadb'
```
mysql version is:

`mysql  Ver 14.14 Distrib 5.7.17, for Linux (x86_64) using  EditLine wrapper`

Django version is: 1.10.5

Please check it, thanks.

**Comment by adamchainz** [comment]
read above, follow install instructions

**Comment by shashanks1** [comment]
Yes I did follow the exact steps for installation and still I am getting error. This is the reason I posted again

**Comment by adamchainz** [comment]
If you have it in your INSTALLED_APPS, this won't happen

**Comment by shashanks1** [comment]
Please this:

```
INSTALLED_APPS = [
    'oauth2_provider',
    'rest_framework',
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'corsheaders',
    'bootstrap3',
    'crispy_forms',
    'django_mysql'
]
```

This is how I include the django_mysql in my apps, if I did something wrong here then please correct.

**Comment by adamchainz** [comment]
Put `django_mysql` at the top, that might fix it

**Comment by adamchainz** [comment]
@shashanks1  did you manage to fix it?

**Comment by shashanks1** [comment]
No I tried to move the app on top in installed apps and it does not work for me. So I left for another option.

**Comment by adamchainz** [comment]
@shashanks1 if you can paste the full stack trace in #357 that would be very useful. I suspect something is managing to run code that checks for `is_mariadb` before our `app.ready` has been called and monkey patched it on. Full fix will be #323.
